gini/gini-vision-lib-ios
# Multipage feature toggle

Since the multipage feature will be optional, it has been added a toggle to enable/disable it along with different check in all multipage related screens.

### How to test
Run the app and try to use the multipage feature. Go to Setting from the API selection screen and disable the multipage feature. After that try to use the multipage and see that it is not possible.

### Merging
Automatic

Issue #172
